lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -419,10 +534,33 @@ sub add_api_user {
     if ( $allow ) {
         $result = 1 if ( $self->{db}->add_api_user( $p->{username}, $p->{api_key} ) eq '1' );
     }
-    
+
     return $result;
 }
 
+$json_schemas{add_batch_job} = joi->object->strict->props(
+    username => $zm_validator->username->required,
+    api_key => $zm_validator->api_key->required,
+    domains => joi->array->strict->items(
+        $zm_validator->domain_name->required
+    )->required,
+    test_params => joi->object->strict->props(
+        ipv4 => joi->boolean,
```

Comment:
Could you update the order of the parameters, either here or in the API documentation or both, so that they match up?
And also add the missing `advanced`, `priority` and `queue` parameters here.

Reply:
Done